resources/views/cruds/fields/defunct_choice.blade.php

```diff
@@ -0,0 +1,8 @@
+<div class="form-group">
+    <label for="is_defunct">{{ trans('organisations.fields.is_defunct') }}</label>
```

Comment:
Note, whenever I work on views, I try and update the `trans()` function calls to `__()` because __() also looks in json translation files (which we don't currently use but might in the future). So please replace trans with __ :)